tests/storage/hotplug.go

```diff
@@ -602,6 +602,64 @@ var _ = SIGDescribe("Hotplug", func() {
 			Entry("DataVolume", addDVVolumeVM, removeVolumeVM),
 			Entry("PersistentVolume", addPVCVolumeVM, removeVolumeVM),
 		)
+
+		It("Should preserve access to block devices if virt-handler crashes", func() {
+			blockDevices := []string{"/dev/disk0"}
+			By("Adding a hotplug block volume")
+			dv := createDataVolumeAndWaitForImport(sc, corev1.PersistentVolumeBlock)
+			blockDevices = append(blockDevices, fmt.Sprintf("/var/run/kubevirt/hotplug-disks/%s", dv.Name))
+			addDVVolumeVM(vm.Name, vm.Namespace, dv.Name, dv.Name, v1.DiskBusSCSI, false, "")
+			By("Verifying the volume has been added to the template spec")
+			verifyVolumeAndDiskVMAdded(virtClient, vm, dv.Name)
+			By("Starting the VM")
+			vm = tests.StartVirtualMachine(vm)
+			vmi, err := virtClient.VirtualMachineInstance(vm.Namespace).Get(vm.Name, &metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			By("Verifying the volume is attached and usable")
+			verifyVolumeAndDiskVMIAdded(virtClient, vmi, dv.Name)
+			verifyVolumeStatus(vmi, v1.VolumeReady, "", dv.Name)
+			getVmiConsoleAndLogin(vmi)
+			targets := verifyHotplugAttachedAndUseable(vmi, []string{dv.Name})
+			Expect(targets).To(HaveLen(1))
+			By("Deleting virt-handler pod")
+			virtHandlerPod, err := kubecli.NewVirtHandlerClient(virtClient).
+				Namespace(flags.KubeVirtInstallNamespace).
+				ForNode(vmi.Status.NodeName).
+				Pod()
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(func() bool {
+				err := virtClient.CoreV1().
+					Pods(virtHandlerPod.GetObjectMeta().GetNamespace()).
+					Delete(context.Background(), virtHandlerPod.GetObjectMeta().GetName(), metav1.DeleteOptions{})
+				return errors.IsNotFound(err)
+			}, 60*time.Second, 1*time.Second).Should(BeTrue())
```

Comment:
nit:
```suggestion
			}, 60*time.Second, 1*time.Second).Should(BeTrue(), "virt-handler Pod is expected to be deleted")
```

Otherwise the error message would look something like "false expected to be true". Would you mind doing that for the other `Eventually` occurrences too?